src/main/java/edu/wpi/first/pathweaver/spline/wpilib/WpilibSpline.java

```diff
@@ -65,13 +65,14 @@ public void update() {
         for (int i = 1; i < waypoints.size(); i++) {
             Waypoint segStart = waypoints.get(i - 1).copy();
             Waypoint segEnd = waypoints.get(i).copy();
+            QuinticHermiteSpline quintic;
 
             if (segStart.isReversed()) {
-                segStart.reverseTangent();
-                segEnd.reverseTangent();
+                quintic = getQuinticSplinesFromWaypoints(new Waypoint[]{segEnd, segStart})[0];
```

Comment:
I thought we decided not to reverse the waypoints?

Reply:
This is just for drawing the lines between them.  Because the program currently only draws lines out the "first" and into the "last".  I believe this doesn't actually affect waypoint order like the previous change did.